docs/deploy-from-windows.md

```diff
@@ -17,17 +24,25 @@ Wait a minute for Windows to install a few things in the background (it will eve
 5. Restart Windows and then [install Ubuntu from the Windows Store](https://www.microsoft.com/p/ubuntu/9nblggh4msv6).
 6. Run Ubuntu from the Start menu. It will take a few minutes to install. It will have you create a separate user account for the Linux subsystem. Once that's done, you will finally have Ubuntu running somewhat integrated with Windows.
 
+## Install Algo
 
-Install additional packages:
+Run these commands in the Ubuntu Terminal to install a prerequisite package and download the Algo scripts to your home directory. Note that when using WSL you should **not** install Algo in the `/mnt/c` directory due to problems with file permissions.
+
+You may need to follow [these directions](https://devblogs.microsoft.com/commandline/copy-and-paste-arrives-for-linuxwsl-consoles/) in order to paste commands into the Ubuntu Terminal.
 
 ```shell
-sudo apt-get update && sudo apt-get install git build-essential libssl-dev libffi-dev python3-dev python3-pip python3-setuptools python3-virtualenv -y
+cd
+umask 0002
+sudo apt update
+sudo apt install -y python3-virtualenv
+git clone https://github.com/trailofbits/algo
+cd algo
 ```
 
-Clone the Algo repository:
+Now you can continue by following the [README](https://github.com/trailofbits/algo#deploy-the-algo-server) from the 4th step to deploy your Algo server!
 
+You'll be instructed to edit the file `config.cfg` in order to specify the Algo user accounts to be created. If you're new to Linux the simplest editor to use is `nano`. To edit the file while in the `algo` directory, run:
 ```shell
-cd ~ && git clone https://github.com/trailofbits/algo && cd algo
+nano config.cfg
```

Comment:
Can we use here `editor` instead of `nano`?

Reply:
I think hiding the true name of the program from new users will hinder them if they need to Google for help. And anyone who knows enough to have used `update-alternatives` will likely disregard this suggestion anyway.

Reply:
`nano` worked fine for me.